index.d.ts

```diff
@@ -29,6 +29,7 @@ export class Logger {
   error(context: Error): void;
   enable(): void;
   disable(): void;
+  drain(timeout?: number): Promise<void>;
```

Comment:
I considered `drain({ timeout })` but decided against it because I do not foresee any additional options